### PR TITLE
[All hosts] (NAA) Latest NAA updates

### PR DESCRIPTION
**Pull request by davidchesnut**
This updates the walkthrough and FAQ with latest updates around NAA preview and GA.

### CODE REVIEW

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following steps show how to enable NAA in the `taskpane.js` or `taskpane.ts`
 
     ```json
     "dependencies": {
-        "@azure/msal-browser": "^3.15.0",
+        "@azure/msal-browser": "^3.24.0",
         ...
```
Comment (AlexJerabek) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:65]: Any idea why this is showing up weird in GitHub?

image

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 2]
```diff
@@ -237,35 +237,35 @@ The following table shows which APIs are supported when NAA is enabled in the MS
...
-| _loginRedirect_               | NO (throws exception) |
-| _logout_                      | NO (throws exception) |
-| _logoutPopup_                 | NO (throws exception) |
-| _logoutRedirect_              | NO (throws exception) |
-| _removeEventCallback_         | YES                   |
-| _removePerformanceCallback_   | NO (throws exception) |
-| _setActiveAccount_            | NO                    |
-| _setLogger_                   | YES                   |
-| _ssoSilent_                   | YES                   |
+| *acquireTokenByCode*          | NO (throws exception) |
```
Comment (AlexJerabek) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:240]: Instead of italic, should these method names be in code ticks?

Comment (AlexJerabek) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:240]: Can we change the "YES" and "NO" to "Yes" and "No"?

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md [hunk 3]
```diff
@@ -59,7 +62,7 @@ To find out whether your add-in uses legacy Exchange user identity tokens and ca
 
 If your add-in calls any of these APIs, you should adopt NAA and migrate to using Entra ID tokens to access Microsoft Graph instead.
 
-We'll provide tooling via PowerShell for Microsoft 365 administrators in October 2024 to turn legacy Exchange tokens on or off in your tenant. This will allow you to test if any add-ins are using Exchange tokens. We'll provide more information when the tooling is ready in this FAQ.
+We'll provide tooling via PowerShell for Microsoft 365 administrators in October 2024 to report the app ID of any add-in that used legacy Exchange online tokens in the last 28 days. The tooling will also allow you to turn legacy Exchange online tokens on or off for the entire tenant. We'll provide more information when the tooling is ready in this FAQ.
```
Comment (AlexJerabek) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md:65]: Since it's October, do we know when this tooling is coming?

**Review by AlexJerabek — APPROVED** [review]
Approving with some suggestions.

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md (hunk 2 above)
Comment (davidchesnut) [reply docs/develop/enable-nested-app-authentication-in-your-add-in.md:244]: fixed

Comment (davidchesnut) [reply docs/develop/enable-nested-app-authentication-in-your-add-in.md:244]: fixed

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md (hunk 1 above)
Comment (davidchesnut) [reply docs/develop/enable-nested-app-authentication-in-your-add-in.md:60]: no it's odd.

**Pull request merged**